index.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+entries:
+  metrics-server-exporter:
+  - apiVersion: v1
+    appVersion: 0.0.7
+    created: "2021-05-28T14:07:57.63680224+02:00"
+    description: A Helm chart for metrics-server-exporter
+    digest: 5d0871e4155cb739e01858d2300da7708b69cad7f77a21bdd6e5584e23ae1f90
+    name: metrics-server-exporter
+    urls:
+    - https://github.com/SplashThat/metrics-server-exporter/blob/mag/create-repo/pkg/metrics-server-exporter-0.1.0.tgz
```

Comment:
when this gets merged should we change the ref to master?

Reply:
indeed, I'd like to avoid that point. This is just to make it work.
The real way of working is creating a helm repository with github-pages with these instructions:
https://medium.com/@mattiaperi/create-a-public-helm-chart-repository-with-github-pages-49b180dbb417